live_site.egg-info/requires.txt

```diff
@@ -0,0 +1,5 @@
+click==8.0.1
+requests>=2.25.1
```

Comment:
we have a whole bunch of these egg info files - but we probably dont want them checked into git (unless we do? Do you know what they're for?)

To fix this problem, we use the .gitignore file - https://www.atlassian.com/git/tutorials/saving-changes/gitignore

.gitignore is checked in to the repo, and it tells git what files to ignore (so the name kinda makes sense haha). If you put the correct entry into the gitignore file, these won't show up in the list of changes

Now that they've been checked in, we might have to do some more surgery to get them out - but stuff we don't want in the repo should be going into the .gitignore file

Reply:
If this egg info helps you maintain the python environment, maybe it is a good idea - I just dont know enough about it, so you'll have to explain it to me